### PR TITLE
edit student fixes -> pen match logic and some ui changes

### PR DESCRIPTION
**Pull request by alexmcdermid**
- add space before icon
- fix logic for what assigned pen and tooltip to show
- change fill type and color of icon
- step 3 pen column to assigned pen and show under review if no assigned pen in table
- custom tooltip width instead of max length

### CODE REVIEW

File: frontend/src/components/common/CustomTable.vue [hunk 1]
```diff
@@ -250,6 +265,13 @@ export default {
         return 'Info Warning';
       }
     },
+    getAssignedPen(assignedPen) {
+      if (assignedPen) {
+        return assignedPen;
+      } else {
+        return 'under review';
```
Comment (arcshiftsolutions) [inline frontend/src/components/common/CustomTable.vue:272]: Can you please capitalize? `Under review`

File: frontend/src/components/common/EditStudent.vue [hunk 2]
```diff
@@ -768,17 +767,17 @@ export default {
         return '';
       }
     },
-    getAssignedPen(assignedPen, studentPen){
-      if (!assignedPen) {
+    getAssignedPen(assignedPen, studentPen, penMatchResult){
+      if (!assignedPen && !penMatchResult) {
         return 'waiting on fixes';
```
Comment (arcshiftsolutions) [inline frontend/src/components/common/EditStudent.vue:772]: Capitalize please

File: frontend/src/components/common/EditStudent.vue [hunk 3]
```diff
@@ -768,17 +767,17 @@ export default {
...
       } else if (assignedPen === studentPen) {
         return assignedPen;
       } else {
```
Comment (arcshiftsolutions) [inline frontend/src/components/common/EditStudent.vue:775]: Caps

File: frontend/src/components/common/CustomTable.vue (hunk 1 above)
Comment (alexmcdermid) [reply frontend/src/components/common/CustomTable.vue:272]: @arcshiftsolutions I'd be happy to but it is not this way in the user story, it is lower case

File: frontend/src/components/common/EditStudent.vue [hunk 4]
```diff
@@ -768,17 +767,17 @@ export default {
...
         return 'under review';
       }
     },
-    getAssignedPenTooltip(assignedPen, studentPen){
-      if (!assignedPen) {
+    getAssignedPenTooltip(assignedPen, studentPen, penMatchResult){
+      if (!assignedPen && !penMatchResult) {
         return 'The submitted student details have errors or incomplete information. Confirm the submitted student name and date of birth.';
       } else if (assignedPen === studentPen) {
         return 'Differences between the Assigned PEN and Submitted PEN indicate an existing student file has been matched to the submitted details. The Assigned PEN will be used to prevent duplication.';
```
Comment (arcshiftsolutions) [inline frontend/src/components/common/EditStudent.vue:783]: This message is questionable; if the assigned PEN and submitted PEN match, I don't get the start of the sentence.

File: frontend/src/components/common/EditStudent.vue (hunk 2 above)
Comment (alexmcdermid) [reply frontend/src/components/common/EditStudent.vue]: You want PenMatchResult? Most things in the file are camelCase

**Review by arcshiftsolutions — APPROVED** [review]

**Pull request merged**